pkgs/development/python-modules/cffi/default.nix

```diff
@@ -30,7 +30,11 @@ if isPyPy then null else buildPythonPackage rec {
   NIX_CFLAGS_COMPILE = lib.optionalString stdenv.cc.isClang
     "-Wno-unused-command-line-argument -Wno-unreachable-code -Wno-c++11-narrowing";
 
-  doCheck = !stdenv.hostPlatform.isMusl;
+  # Lots of tests fail on aarch64-darwin due to "Cannot allocate write+execute memory":
+  # * https://cffi.readthedocs.io/en/latest/using.html#callbacks
+  doCheck = !stdenv.hostPlatform.isMusl && !(stdenv.isDarwin && stdenv.isAarch64);
+
+  pytestFlagsArray = ["--ignore=testing/cffi1/test_parse_c_type.py"];
```

Comment:
This will cause a rebuild

Reply:
This was my first attempt that shouldn't have made it here, removed.